polygeist/tools/cgeist/driver.cc

```diff
@@ -387,7 +387,7 @@ static LogicalResult optimize(mlir::MLIRContext &Ctx,
   if (OptLevel != llvm::OptimizationLevel::O0) {
     PM.addPass(polygeist::createArgumentPromotionPass());
     PM.addPass(polygeist::createKernelDisjointSpecializationPass(
-        {options.getCgeistOpts().getRelaxedAliasing()}));
+        {options.getCgeistOpts().getRelaxedAliasing(), UseOpaquePointers}));
```

Comment:
We could create a variable holding `{options.getCgeistOpts().getRelaxedAliasing(), UseOpaquePointers})` rather than repeating the same options every time we add a pass.